src/types/models.ts

```diff
@@ -87,14 +87,14 @@ export interface Organization {
 	is_using__billing_version: 'v1' | 'v2';
 	logo_image: WebResource;
 
-	application: ReverseNavigationResource<Application>;
+	application?: ReverseNavigationResource<Application>;
```

Comment:
I made all the reverse navigation props optional as the improved typings now require that since they are usually omitted (ie whenever they aren't explicitly expanded)